sample_xblocks/basic/test/test_view_counter.py

```diff
@@ -1,20 +1,17 @@
 """ Simple test for the view counter that verifies that it is updating properly """
 
-from collections import namedtuple
-
 from mock import Mock
 from xblock.runtime import DictKeyValueStore, KvsFieldData
-from xblock.test.tools import TestRuntime, assert_equals, assert_in
+from xblock.test.tools import TestRuntime as TR
```

Comment:
I'm curious, what prompted using this alias?

Reply:
Pytest was trying to collect it as a test due to the name and was throwing a warning about it having an __init__. It should really be renamed in xblock, but I'm already pretty far afield in doing this PR. I'll comment it to explain.